src/formula_gen.cpp

```diff
@@ -260,10 +283,6 @@ bool generateSingle(const Program& p, Formula& result, bool pariMode) {
       return false;
     }
     // TODO: remove this limitation
```

Comment:
What is this limitation about?

Reply:
This limitation forbids access to the loop counter cell in simple loops (IE programs). Here is an example:

```asm
; A000142: Factorial numbers: n! = 1*2*3*4*...*n (order of symmetric group S_n, number of permutations of n letters).

mov $1,1
lpb $0       ; $0 is the loop counter
  mul $1,$0  ; $1 is "loop-counter dependent" because its value depends on $0
  sub $0,1
lpe
mov $0,$1
```
Removing this limitation means, we can generate recursive formulas that not only access previous terms `a(n-1)` but also `n` itself. For example `A000142: a(n) = n*a(n-1), a(0) = 1`